### PR TITLE
Hand signals enhancement

### PR DESCRIPTION
**Pull request by jokoho48**
Port THS to ACE3 after #865

### CODE REVIEW

File: addons/handSignals/stringtable.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="Arma3">
+    <Package name="ACE_HANDSIGNALS">
+        <Key ID="STR_ACE_HANDSIGNALS_NAME">
+            <English>ACE Hand Signals</English>
+            <German>ACE Hand Signale</German>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>ACE Sygnały ręką</Polish>```

File: addons/handSignals/stringtable.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        </Key>
+        <!-- STOP -->
+        <Key ID="STR_ACE_HANDSIGNALS_STOP">
+            <English>Stop</English>
+            <French>Stop</French>
+            <German>Stop</German>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Stop</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        </Key>
+
+        <!-- COVER -->
+        <Key ID="STR_ACE_HANDSIGNALS_COVERS">
+            <English>Get cover</English>
+            <French>À couvert</French>
+            <German>Deckung Suchen</German>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Do osłon</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        </Key>
+
+        <!-- REGROUP -->
+        <Key ID="STR_ACE_HANDSIGNALS_REGROUP">
+            <English>Rally up</English>
+            <French>Regroupement</French>
+            <German>Sammeln</German>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Zbiórka</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        </Key>
+
+        <!-- FORWARD -->
+        <Key ID="STR_ACE_HANDSIGNALS_FORWARD">
+            <English>Move forward</English>
+            <French>En avant</French>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Naprzód</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        <!-- ENGAGE -->
+        <Key ID="STR_ACE_HANDSIGNALS_ENGAGE">
+            <English>Engage</English>
+            <French>Engager</French>
+        </Key>
+
+        <!-- POINT -->
+        <Key ID="STR_ACE_HANDSIGNALS_POINT">
+            <English>Point</English>
+            <French>Pointer</French>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Wskaż</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        <Key ID="STR_ACE_HANDSIGNALS_FORWARD">
+            <English>Move forward</English>
+            <French>En avant</French>
+            <German>Vorwärts Bewegen</German>
+        </Key>
+
+        <!-- ENGAGE -->
+        <Key ID="STR_ACE_HANDSIGNALS_ENGAGE">
+            <English>Engage</English>
+            <French>Engager</French>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Atak</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+            <English>Point</English>
+            <French>Pointer</French>
+            <German>Zeigen</German>
+        </Key>
+
+        <!-- HOLD -->
+        <Key ID="STR_ACE_HANDSIGNALS_HOLD">
+            <English>Hold</English>
+            <French>Tenir</French>
+            <German>Anhalten</German>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Wstrzymać</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+        </Key>
+
+        <!-- WARNING -->
+        <Key ID="STR_ACE_HANDSIGNALS_WARNINGS">
+            <English>Warning</English>
+            <French>Attention</French>
+            <German>Achtung</German>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Uwaga</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+        </Key>
+        <!-- FREE LOOK -->
+        <Key ID="STR_ACE_HANDSIGNALS_FREELOOK">
+            <English> &#43; freelook</English>
+            <French> &#43; vue libre</French>
+            <German> &#43; Frei Blick</German>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish> &#43; rozglądanie się</Polish> ```

File: addons/handSignals/stringtable.xml [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+        </Key>
+		<!-- Display Name for ACE Interaction -->
+        <Key ID="STR_ACE_HANDSIGNALS_DISPLAYNAME">
+            <Original>Extra Gestures</Original>
+            <German>Exta Gesten</German>
```
Comment (gienkov) [inline addons/handSignals/stringtable.xml]: ```<Polish>Gesty dodatkowe</Polish>```

Comment (gienkov) [inline addons/handSignals/stringtable.xml]: You should change that from Original to English.

**Comment by jonpas** [comment]
Per discussion on Slack, I see no reason to have 2 separate modules (PBOs) for gestures, currently there are also 2 submenus, one for current gestures and one for these. I'd say it should all be in one PBO and spread into sublevels per function (however that can be classified). I feel like a lot if not most of the current gestures could be removed with this.

**Comment by alganthe** [comment]
Same train of thoughs as @jonpas here.

**Pull request closed without merging**